### PR TITLE
DocUpdate for Testnet p2

### PR DESCRIPTION
**Pull request by mohammedpatla**
Changed typo:
line 73 command - from a -> from <key-alias>

### CODE REVIEW

File: docs/testnet/deploy-contract.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ Example parameters:
 Now, to initialize the contract we will use the following command (replace the parameters with values of your choosing): 
 
 ```
-secretcli tx compute instantiate <code_id> --label <choose-an-alias> '{"name": "<coin_name>", "symbol": "<coin_symbol>", "decimals": <num_of_decimals>, "initial_balances": []}' --from a
+secretcli tx compute instantiate <code_id> --label <choose-an-alias> '{"name": "<coin_name>", "symbol": "<coin_symbol>", "decimals": <num_of_decimals>, "initial_balances": []}' --from <key-alias>
 ```
```
Comment (reuvenpo) [inline docs/testnet/deploy-contract.md:74]: On line 24 of this file, the terminology `--from <wallet_name>` is used.
For consistency's sake, these should be the same.
Can you please also change line 24 accordingly?

**Comment by reuvenpo** [comment]
Thanks for the PR Mohammed! :)

**Comment by assafmo** [comment]
Thanks you!

File: docs/testnet/deploy-contract.md (hunk 1 above)
Comment (mohammedpatla) [reply docs/testnet/deploy-contract.md:74]: Whoops, already merged @reuvenpo

Comment (reuvenpo) [reply docs/testnet/deploy-contract.md:74]: yeah, no biggie, I wanted to give you an opportunity to get feedback on your PR, but we decided it's best to just accept it as is right now, and make additional fixes later. If you feel like it, you're more than welcome to submit more PRs with edits and corrections :smile:

**Pull request merged**